lib/matplotlib/testing/__init__.py

```diff
@@ -50,39 +50,28 @@ def setup():
     set_reproducibility_for_testing()
 
 
-def subprocess_run_helper(func, *args, timeout, **extra_env):
+def subprocess_run_helper(func, *args, timeout, env=None):
```

Comment:
```suggestion
def subprocess_run_helper(func, *args, timeout, extra_env=None):
```
Or modify the docs.

Should there be a default value for timeout?

Reply:
Better change the docs (as `env` is used in the code).

Reply:
extra_env does seem like a nicer name, IMO.
I think forcing an explicit timeout for tests is not a bad idea...

Reply:
I was just a bit confused about how that worked in practice (probably due to not knowing the details about Python here). Will the last non-keyword argument be `timeout`? If so, it is provided in the calls just for readability? Or is this a way to get a keyword only argument without a default?

(Just out of curiosity, so no need to answer.)

Reply:
It's a keyword-only argument with no default, i.e. you *need* to pass `timeout=something` to all calls.